### DISCUSSION TITLE
Official Project Name That Is More Meaningful Than Distribution!

### DISCUSSION THREAD

**Discussion by Vad1mo** [distribution/distribution#4255] [Polls]
This project was renamed from **Docker Distribution** to only distribution. This makes sense as it is not purely Docker related anymore. However, calling it **"Distribution"**, is also misleading. It makes it impossible to reference or talk about the project correctly.

But the current name, **"Distribution"**, is misleading and not precises. I am not arguing about the GitHub URL, just the name of the project, as it makes no sense to call it distribution. Distribution of what?

So I propose a few names, I am also open to other names too.

This proposal is not abut changing the GH URL, only the human-readable and understandable naming of the project. 
My proposal also only considers a name which is backwards compatible, meaning when the docs or other sources are talking about "Distribution" it will not become wrong to keep referencing it in that way.

**Comment by sudo-bmitch** [comment]
I don't think any of these names are correct since this project will be lagging the OCI spec and is not hosted by the OCI organization. It also risks confusion with the OCI distribution-spec. I'd recommend referring to it as the CNCF Distribution Project.

**Reply by Vad1mo** [reply]
Appreciate your feedback @sudo-bmitch.
 
I updated, the poll and added **CNCF Distribution Project**. Makes sense to reference CNCF,
Even so, I think, and this is my main criticism with the current name, Distribution should reference what it is distributing; otherwise we are not gaining any clarification.  

> this project will be lagging the OCI spec and is not hosted by the OCI organization
- IMO, it is common that implementations don't implement the full spec (temporarily or permanent). But we can agree that this project does not intend to implement something that contradicts the OCI spec.  
 

>  It also risks confusion with the OCI distribution-spec
- Given the current contextless name and the potential confusion with OCI distribution-spec, I rather prefer the latter, as at least the confusion would be somewhat related. I also don't see any confusion for an engineer to distinguish between spec and implementation.

**Reply by sudo-bmitch** [reply]
> it is common that implementations don't implement the full spec

I believe users would assume otherwise, particularly if it's called the OCI implementation of the spec, and even moreso if it's the reference implementation.

**Comment by brackendawson** [comment]
Echo @sudo-bmitch on use of OCI. This is a CNCF project, it's a Docker registry too, it shouldn't say OCI in the name.

When we need to talk about this repo in IBM we tend to verbalise it as "Distribution Registry". There's also been an effort to move the client API code out of this repo recently, leaving it as literally the registry. So, some suggestions:

* Distribution Registry (github.com/distribution/registry)
* CNCF Container Registry
* CNCF Artifact Registry
